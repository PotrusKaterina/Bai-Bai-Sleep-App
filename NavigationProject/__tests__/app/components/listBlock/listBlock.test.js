import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ListBlock from '../../../../app/components/listBlock/listBlock';


describe('AuthInput', () => {
    test('create snapshot AuthInput', () => {
        const render = new ShallowRenderer();
        const snapshot = render.render(<ListBlock />);
        expect(snapshot).toMatchSnapshot();
    })
});