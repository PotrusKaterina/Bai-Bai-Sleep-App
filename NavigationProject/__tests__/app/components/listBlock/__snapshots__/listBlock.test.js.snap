// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthInput create snapshot AuthInput 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <SectionList
    data={Array []}
    disableVirtualization={false}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onEndReachedThreshold={2}
    renderItem={[Function]}
    renderSectionHeader={[Function]}
    scrollEventThrottle={50}
    sections={Array []}
    stickySectionHeadersEnabled={true}
    updateCellsBatchingPeriod={50}
    windowSize={21}
  />
</View>
`;
